src/test/java/com/bobocode/parser/YamlPropertyParserTest.java

```diff
@@ -0,0 +1,116 @@
+package com.bobocode.parser;
+
+import com.bobocode.exception.PersistenceFileNotFoundException;
+import com.bobocode.exception.PersistenceUnitNotFoundException;
+import org.junit.jupiter.api.Test;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class YamlPropertyParserTest {
+
+    private static final String PERSISTENCE_UNIT_NAME_PROPERTY = "persistence-unit.name";
+    private static final String LOG_LEVEL_PROPERTY = "log-level";
+    private static final String DATA_SOURCE_JDBC_URL_PROPERTY = "persistence-unit.dataSource.jdbcUrl";
+    private static final String DATA_SOURCE_USER_PROPERTY = "persistence-unit.dataSource.user";
+    private static final String DATA_SOURCE_PASSWORD_PROPERTY = "persistence-unit.dataSource.password";
+    private static final String DIALECT_PROPERTY = "persistence-unit.dialect";
+
+    @Test
+    void testReadPropertiesForPersistenceUnit_DefaultFileName() {
```

Comment:
Better use '@Display("...")` and name use short test method names

Reply:
Done